src/predict.jl

```diff
@@ -3,7 +3,8 @@
     predict(estimator, X; [predict_type = 0, num_iterations = -1, verbosity = 1,
     is_row_major = false])
 
-Return an array with the labels that the `estimator` predicts for features data `X`.
+Return a **MATRIX** with the labels that the `estimator` predicts for features data `X`.
+Use `dropdims` if a vector is required.
```

Comment:
How about a return type sig just to make this clear?

Reply:
Ever since i discovered that adding a return type makes julia call convert on the output, I'm a lot less eager to do o, although I'll do it anyway

Reply:
Yes, I think it is generally better to write type-stable code that makes these type annotations on return types unnecessary. Of course comments indicating the type are helpful.

Reply:
e99287d